shared/css/top/top.css

```diff
@@ -38,11 +38,61 @@ body {
   word-wrap: break-word;
 }
 
+#post_content {
+    background: white;
+    border-radius: 6px;
+    padding: 13px;
+    box-shadow: 4px 5px 0px #8E8980;
+}
+
+#post_content p {
+  margin: 0;
+  word-wrap: break-word;
+}
+
+#cleartext {
+  height: 100%;
+}
+
+@media(min-width: 992px){
+  #dashed_hr {
```

Comment:
I'll run this on my computer, but I think this should be tied to a responsive value from [bootstrap](http://getbootstrap.com/). I'm not sure what that is at the moment.

@spectralsun do you know how to trigger this CSS rule when bootstrap makes the columns stack vertical?